Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rust:latest AS builder
```

Comment:
Extend from `debian:slim`?

Reply:
Fixed in [4cae812](https://github.com/kiesraad/abacus/pull/429/commits/4cae812f9ae66af4fd92505dfa9544e003849eee)